### ISSUE TITLE
generate approximate string matching query using pypher

### ISSUE THREAD

**Issue by rameshjesswani** [emehrkay/Pypher#20]
I am trying to create neo4j query : 
``` match (n) where n.name contains 'per' return n```

 but i am unable to find function similar to ```contains``` in this library. Can anyone please help me this?

**Comment by emehrkay** [comment]
I am on my phone and cannot test it, but you should be able to do `.CONTAINS(‘per’)` (all caps because it is a python statement)

**Comment by rameshjesswani** [comment]
@emehrkay can you please write the complete query: This is query when exact name of 
node is defined
```
self.pypherObject.Match.node('u').WHERE.u.property('name') == 'person'
```

**Comment by emehrkay** [comment]
I’d write your original query like this:

from pypher.builder import Pypher, Param
p = Pypher()
p.MATCH.node('n').where.n.CONTAINS(Param('per_param', 'per')).RETURN.n

**Comment by emehrkay** [comment]
I'm at a computer now and there was an error in the query that I posted, try this:

```python
p.MATCH.node('n').where.n.__name__.CONTAINS(Param('per_param', 'per')).RETURN.n
```

![screen shot 2018-10-17 at 4 16 41 pm](https://user-images.githubusercontent.com/24242/47113859-23720580-d228-11e8-911a-b148b2a9a87b.png)

**Comment by rameshjesswani** [comment]
@emehrkay  I have tried this, but it does not return correct cypher query:

```
MATCH (u) WHERE u.`name` CONTAINS <pypher.builder.Param object at 0x7f5a240d6048> RETURN u
```

Param object does not return  any parameters

**Comment by emehrkay** [comment]
Can you post your code? Are you using the latest version?

**Comment by rameshjesswani** [comment]
pypher version: ```0.13.0```

code:

```
p = Pypher()
p.MATCH.node('n').where.n.__name__.CONTAINS(Params('per_param', 'per')).RETURN.n
print (str(p))
```

**Comment by emehrkay** [comment]
Try Param, you have Params

**Comment by rameshjesswani** [comment]
I tried Param as well as Params. but still Param does not returns anything

**Comment by emehrkay** [comment]
It is the version that you're running. I fixed that bug in 0.14.2 

https://github.com/emehrkay/Pypher/blob/master/CHANGELOG.md#0142----9292018

**Comment by rameshjesswani** [comment]
Thanks, yes it was version problem. :)
